src/rental.rb

```diff
@@ -2,7 +2,9 @@ class Rental
   attr_accessor :date

```

Comment:
- [ ] Great job so far 👍 you are almost there. Your `attr_accessor` here should have in addition to `:date`, a `:book` and a `:person`. Kindly rectify this issue